.github/dependabot.yml

```diff
@@ -1,22 +1,18 @@
 version: 2
 updates:
-- package-ecosystem: bundler
-  directory: "/"
-  allow:
-    - dependency-type: "all"
-  schedule:
-    interval: daily
-    time: "21:00"
-    timezone: Europe/London
-  open-pull-requests-limit: 5
-  target-branch: main
-  groups:
-    rswag-gems:
-      patterns:
-        - "rswag-*"
-    aws-gems:
-      patterns:
-        - "aws-*"
-    rubocop-gems:
-      patterns:
-        - "rubocop-*"
+  - package-ecosystem: "bundler"
+    directory: "/"
+    allow:
+      - dependency-type: "all"
+    schedule:
+      interval: weekly
+      day: monday
+      time: "03:00"
```

Comment:
This needs to stay at 9pm - it calls the database ti run tests and it will be offline at 3am